include/materials/mohr_coulomb.tcc

```diff
@@ -264,6 +267,9 @@ void mpm::MohrCoulomb<Tdim>::compute_df_dp(
   const double dtheta_dr = -1 / (3. * std::sqrt(divider));
   // Compute dJ2 / dSigma
   Vector6d dj2_dsigma = dev_stress;
+  dj2_dsigma(3) *= 2.0;
```

Comment:
I am currently rechecking everything. So please wait for me.

Reply:
Sure, @ezrayst, take your time. Thanks for reconfirming.

Reply:
Can we approve this request first, because this is real an error in computation.

Reply:
Ping @ezrayst

Reply:
I can OK it, but maybe we need to revisit it again for sure. What do you think?

Reply:
I mean at least we need to make it correct first and good for other users, and then improve it.

Reply:
We don't know whether it's correct yet, but at least we can fix what is for sure wrong now. I approved it.